src/config.rs

```diff
@@ -99,6 +103,7 @@ mod tests {
 [mastodon]
 delete_older_statuses = true
 delete_older_favs = true
+ignore_boosts = true
```

Comment:
we should test without this new option to make sure we don't break compatibility with existing config files. Probably best to add a new test with the options to also make sure that works?